WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -1084,6 +1088,7 @@ extension EditableOrderViewModel {
              shippingTotal: String = "0",
              shippingMethodTitle: String = "",
              shippingMethodTotal: String = "",
+             shippingTax: String = "",
```

Comment:
The `showShippingTax` property is missing in the struct's initializer signature, so we only get the memberwise one :

```suggestion
             shippingTax: String = "",
             shouldShowShippingTax: Bool = false,
```

With this, should we add a some tests to `EditableOrderViewModelTests` for these properties? Perhaps additional assertions to existing tests would be enough.

Nit: for consistency with other properties in the same struct, should we rename this to `shouldShowShippingTax`? wdyt?

Reply:
I did not add it to init signature because it is set depending on `shippingTax`. I could add it as `Bool?` and then if it is set and true use it, if not set it depending on `shippingTax`